index.js

```diff
@@ -0,0 +1,122 @@
+const todoInput = document.querySelector('.new-todo');
+const todoCount = document.querySelector('.todo-count strong');
+const activeTodosBtn = document.querySelector('.active');
+const completedTodosBtn = document.querySelector('.completed');
```

Comment:
전 `document.querySelector`가 계속 반복되면 큰 의미 없이 코드 전체가 길어진다는 느낌이 들어서 아래와 같이 축약해서 쓰곤합니다 ㅎㅎ

```suggestion
const $ = (selector) => document. querySelector(selector)
const $todoInput = $('.new-todo')
```

Reply:
생각지도 못한 부분이었는데 지적해주셔서 감사합니다!! 앞으로 계속 이렇게 써야겠습니다 👍